File: Tema 03 - Web API/PetShelter/PetShelter.DataAccessLayer/Models/Fund.cs
namespace PetShelter.DataAccessLayer.Models
{
    public class Fund : IEntity
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public decimal DonationAmout { get; set; }
        public decimal Goal { get; set; }
        public DateTime CreationDate { get; set; } = DateTime.Now;
        public DateTime DueDate { get; set; }
        public string Status { get; set; }
        public int? OwnerId { get; set; }

        public Person Owner { get; set; }
    }
}

File: Tema 06 - React/petshelter.ui/src/Models/Fund.ts
import { Person } from "./Person";

export class Fund {
    id: number;
    name: string;
    donationAmout: number;
    goal: number;
    creationDate: Date;
    dueDate: Date;
    status: string;
    owner?: Person;

    constructor(id: number, name: string, donationAmout: number, goal: number, creationDate: Date, dueDate: Date, status: string, owner?: Person) {
        this.id = id;
        this.name = name;
        this.donationAmout = donationAmout;
        this.goal = goal;
        this.creationDate = creationDate;
        this.dueDate = dueDate;
        this.status = status;
        this.owner = owner;
    }
}